packages/style-engine/class-wp-style-engine-css-declarations.php

```diff
@@ -127,15 +127,31 @@ public function get_declarations_string( $should_prettify = false, $indent_count
 		$indent              = $should_prettify ? str_repeat( "\t", $indent_count ) : '';
 		$suffix              = $should_prettify ? ' ' : '';
 		$suffix              = $should_prettify && $indent_count > 0 ? "\n" : $suffix;
+		$spacer              = $should_prettify ? ' ' : '';
 
 		foreach ( $declarations_array as $property => $value ) {
-			$spacer               = $should_prettify ? ' ' : '';
```

Comment:
Moved the `$spacer` outside the loop 'cause it doesn't need to be defined for each iteration of the loop, its value remains constant.